framework/include/materials/CoupledGradientMaterial.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Material.h"
+
+/**
+ * A material that optinally computes two properties, one corresponding to the value of the coupled
```

Comment:
I dont think it does two properties.
I think it only compute the gradient one?

Reply:
crap, those are comments from the original header.